staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -169,9 +169,8 @@ type StatefulSetSpec struct {
 	Replicas *int32 `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`
 
 	// selector is a label query over pods that should match the replica count.
-	// If empty, defaulted to labels on the pod template.
+	// Must be specified that matches the labels present on the pod template.
```

Comment:
/s/"Must be specified that matches the labels present on the pod template"/"It must match the pod template's labels."

Reply:
@kow3ns addressed, PTAL